student_explorer/settings.py

```diff
@@ -156,7 +156,10 @@
 EMAIL_USE_TLS = config('DJANGO_EMAIL_USE_TLS', default=None)
 EMAIL_USE_SSL = config('DJANGO_EMAIL_USE_SSL', default=None)
 
-FEEDBACK_EMAIL = config('DJANGO_FEEDBACK_EMAIL', default=None)
+FEEDBACK_EMAIL = config('DJANGO_FEEDBACK_EMAIL', default='aanant@umich.edu')
```

Comment:
Just leave this one default none
```suggestion
FEEDBACK_EMAIL = config('DJANGO_FEEDBACK_EMAIL', default=None)
```

Reply:
You should be able to test this by changing the value in the .env rather than changing it here.